### ISSUE TITLE
Deactivate vibration when charging option?

### ISSUE THREAD

**Issue by Bard09** [Rytiggy/Glance#41]
It's often a surprise to see your charging Fitbit vibrating...especially if you've been charging in another room and aren't aware it's going off and bothering someone.  Being able to deactivate vibration during charging would be useful to me in this scenario at the very least :-)

**Comment by Rytiggy** [comment]
I plan to add time based alerts in a future update, but in the mean time in Glances settings you can click the disable alerts toggle :)

**Comment by bmammon** [comment]
I agree, I would also use this feature. I believe the Marclock watchface has a toggle for it.
![ada294e1-a2a9-4612-b27c-cb8c7d7d24a8](https://user-images.githubusercontent.com/26399567/51525412-22cbab00-1de5-11e9-884f-a5069db640aa.jpeg)

Another option: Not have the alerts go off indefinitely until pressing snooze. Allow users to customize how long the alert is triggered (how many sets of vibrations) before automatically snoozing itself. Time based alerts would be great for this feature because it would be nice to have more sets of vibrations at night (to help wake up from a low) but less during the day.

**Comment by StormeHawke** [comment]
Consider this my vote and stamp of approval on this issue! :)
